packages/embeds/embed-core/src/preview.ts

```diff
@@ -1,10 +1,10 @@
 // We can't import @calcom/lib/constants here yet as this file is compiled using Vite
-const WEBAPP_URL = process.env.EMBED_PUBLIC_WEBAPP_URL ?? "";
-const EMBED_LIB_URL = process.env.EMBED_PUBLIC_EMBED_LIB_URL ?? WEBAPP_URL;
-const IS_E2E = process.env.NEXT_PUBLIC_IS_E2E === "1";
+const WEBAPP_URL = process.env.EMBED_PUBLIC_WEBAPP_URL || "";
 if (!WEBAPP_URL) {
   throw new Error("WEBAPP_URL is not set");
 }
+const EMBED_LIB_URL = process.env.EMBED_PUBLIC_EMBED_LIB_URL || WEBAPP_URL;
```

Comment:
Removed Nullish colaesing operator as we want non falsy value. Empty string isn't acceptable.

Normally, env variable is undefined but the way we are defining right now in vitest, we always provide empty string fallback, which is the cause of the issue.